src/main/java/org/hydev/mcpm/client/database/LocalPluginTracker.java

```diff
@@ -21,11 +24,16 @@
  */
 public class LocalPluginTracker {
```

Comment:
I talked with Prof Calvin about this class. Since this class accesses the filesystem it's on the outer layer of CA. This means for other classes to properly use it on lower layers, you'll have to make a `PluginTracker` interface.

```suggestion
public class LocalPluginTracker implements PluginTracker {
```